src/translations/en.json

```diff
@@ -5292,8 +5292,8 @@
               "select-options": {
                 "label": "Select options"
               },
-              "number": {
-                "label": "Number",
+              "number-value": {
+                "label": "Number value",
```

Comment:
Wouldn't be "Numeric value" in correct English?

Reply:
```suggestion
                "label": "Numeric value",
```